test/util/e2e.go

```diff
@@ -26,6 +26,8 @@ import (
 )
 
 const (
+	// E2eTestSleepImageOld is the image used for testing rolling update.
+	E2eTestSleepImageOld = "gcr.io/k8s-staging-perf-tests/sleep:v0.0.3@sha256:00ae8e01dd4439edfb7eb9f1960ac28eba16e952956320cce7f2ac08e3446e6b"
```

Comment:
Sorry for the late. This is just a question. Why do we need to use this outdated image?